devcycle_python_sdk/api/local_bucketing.py

```diff
@@ -272,10 +281,50 @@ def _read_assembly_script_byte_array(self, pointer: int) -> bytes:
         for i in range(data_length):
             ret[i] = data[data_pointer + i]
 
-        return ret
+        return bytes(ret)
 
-    def get_variable_for_user_protobuf(self, params_buffer) -> str:
-        return ""
+    def init_event_queue(self, options_json: str) -> None:
+        with self.wasm_lock:
+            options_addr = self._new_assembly_script_string(options_json)
+            self.initEventQueue(self.wasm_store, self.sdk_key_addr, options_addr)
+
+    def get_variable_for_user_protobuf(
+        self, user: User, key: str, default_value: Any
+    ) -> pb2.SDKVariable_PB:
```

Comment:
I think it might be easier to work with a return value of a `Variable` here, by converting from the `pb2.SDKVariable_PB` inside this method instead of in `DevCycleLocalClient`.

IMO it's better that we quarantine the janky protobuf-generated types as much as possible to a limited part of the codebase :)

Reply:
yeah, i kept waffling.  I think you are right and i'll just commit to doing that

Reply:
done